specification.markdown

```diff
@@ -283,10 +283,14 @@ The EBML Header documents the EBML Schema (also known as the EBML DocType) that
 
 The EBML Header MUST contain a single Master Element with an Element Name of EBML and Element ID of 0x1A45DFA3 (see (#ebml-element)) and any number of additional EBML Elements within it. The EBML Header of an EBML Document that uses an EBMLVersion of 1 MUST only contain EBML Elements that are defined as part of this document.
 
+Elements within an EBML Header can be at most 4 octets long, except for the EBML Element with Element Name EBML and Element ID `0x1A45DFA3` (see (#ebml-element)), which can be up to 8 octets long.
+
 ## EBML Body
 
 All data of an EBML Document following the EBML Header is the EBML Body. The end of the EBML Body, as well as the end of the EBML Document that contains the EBML Body, is reached at whichever comes first: the beginning of a new EBML Header at the Root Level or the end of the file. This document defines precisely which EBML Elements are to be used within the EBML Header, but does not name or define which EBML Elements are to be used within the EBML Body. The definition of which EBML Elements are to be used within the EBML Body is defined by an EBML Schema.
 
+Within the EBML Body, the maximum octet length allowed for any Element ID is set by the EBMLMaxIDLength Element and the maximum octet length allowed for any Element Data Size is set by the EBMLMaxSizeLength Element.
```

Comment:
Maybe add the `EBMLMaxIDLength Element of the EBML Header` (same for EBMLMaxSizeLength). To avoid any confusion.